### DISCUSSION TITLE
Building against TFlite

### DISCUSSION THREAD

**Discussion by jeremiahrose** [coqui-ai/STT#1906] [Q&A] [answered]
Hi there,
I saw the archived conversation here: https://github.com/coqui-ai/STT/discussions/625
I'm wondering if anyone can guide me on how to build Coqui with TensorFlow Lite on an Ubuntu Desktop system? Does the build system support this now?
Thanks!
Jerry

**Comment by reuben** [comment]
The [build docs](https://stt.readthedocs.io/en/latest/BUILDING.html#building-binaries) should have the info you're looking for.

**Comment by jeremiahrose** [comment, marked as answer]
Ah yes, looks like the TFLite instructions are hidden in the Building For Android section.

I was able to get a TFLite build (without compiling for Android) by adding this flag to the Bazel build command:

`--define=runtime=tflite`
